html/includes/functions.inc.php

```diff
@@ -12,6 +12,29 @@
  * @copyright  (C) 2013 LibreNMS Group
  */
 
+function var_isset($v) {
```

Comment:
I introduced the var_ functions after noticing the shitload of php warnings because unset $vars we're being accessed. Var_isset may be a bit over the top, but var_eq and var_get are very useful, IMHO.

-- 
Mark Schouten
Tuxis Internet Engineering
mark@tuxis.nl / 0318 200208

> On 30 Aug 2015, at 16:36, "Daniel Preussker" <notifications@github.com> wrote:
> 
> In html/includes/functions.inc.php:
> 
> > @@ -12,6 +12,29 @@
> >   * @copyright  (C) 2013 LibreNMS Group
> >   */
> >  
> > +function var_isset($v) {
> Are these really necessary? They seem to overly complicate access to $vars
> 
> —
> Reply to this email directly or view it on GitHub.
>